src/js/letters/actions/letters.js

```diff
@@ -92,33 +92,8 @@ export function getMailingAddress() {
           data: responseCopy
         });
       },
-      (response) => {
-        const error = response.errors.length > 0 ? response.errors[0] : undefined;
-        if (error) {
-          if (error.status === '503' || error.status === '504') {
-            // Either EVSS or a partner service is down or EVSS times out
-            return dispatch({ type: BACKEND_SERVICE_ERROR });
-          }
-          if (error.status === '403') {
-            // Backend authentication problem
-            return dispatch({ type: BACKEND_AUTHENTICATION_ERROR });
-          }
-          // All other error codes
-          return Promise.reject(
-            new Error(`vets_address_error_server_get: ${error.status}`)
-          );
-        }
-        return Promise.reject(
-          new Error('vets_address_error_server_get')
-        );
-      }
-    ).catch((error) => {
-      if (error.message.match('vets_address_error_server_get')) {
-        Raven.captureException(error);
-        return dispatch({ type: GET_ADDRESS_FAILURE });
-      }
-      throw error;
-    });
+      () => dispatch({ type: GET_ADDRESS_FAILURE })
```

Comment:
This also means we're not going to log any of these errors to Sentry. Is that okay? They're already logged on the back end, right?

Reply:
Yep, I confirmed with Alastair that they're all being logged on the back-end, so I think we're good.